modules/programs/git.nix

```diff
@@ -201,6 +201,22 @@ in {
         '';
       };
 
+      hooks = mkOption {
```

Comment:
You need to add an option description. Also make sure to add a test case that uses the new option.